src/views/forms/AccountManagementForms/TokenSelectMenu.tsx

```diff
@@ -24,6 +25,11 @@ type ElementProps = {
   isExchange?: boolean;
 };
 
+export const tokenIsCCTPToken = (token: SelectionOption): boolean => {
+  if (!token?.stringKey) return false;
+  return CCTP_TOKEN_NAMES_LOWER_CASE.includes(token.stringKey.toLowerCase());
```

Comment:
update to use token.value

Reply:
decided to get rid of this in favor of the existing address object